spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,16 @@
 RSpec.configure do |config|
-  config.before(:each) do
+  config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
```

Comment:
But you call want to change it to false to not lint the factories. In fact, the ideal solution would be running the linting code in a task as it is said here: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories

> Running it in a before(:suite), will negatively impact the performance of your tests when running single tests.

Then we can run this task in Travis and if it success then running the tests too.
And for the `mock_commercial_request` we already have this `before_each`: https://github.com/openSUSE/osem/blob/c48c8afa51b3072af27ae76bb2ef0523bf1c50a8/spec/support/external_request.rb#L6, so as long as we keep this code it should work. And to be honest I don't know why the `mock_commercial_request` call is inside the if `ENV['OSEM_FACTORY_LINT'] != 'false'` as this is completely unrelated to the lint stuff.